frontend/test/metabase/scenarios/dashboard/dashboard.cy.spec.js

```diff
@@ -771,6 +771,71 @@ describe("scenarios > dashboard", () => {
     cy.get(".Modal--full").should("not.exist");
     assertScrollBarExists();
   });
+
+  it("dashboard filters should allow multiple selections (metabase#15689)", () => {
+    cy.createQuestion({
+      name: "15689",
+      query: { "source-table": PEOPLE_ID },
+    }).then(({ body: { id: QUESTION_ID } }) => {
+      cy.createDashboard("15689D").then(({ body: { id: DASHBOARD_ID } }) => {
+        // Add filter to the dashboard
+        cy.request("PUT", `/api/dashboard/${DASHBOARD_ID}`, {
+          parameters: [
+            {
+              name: "Location",
+              slug: "location",
+              id: "5aefc725",
+              type: "string/=",
```

Comment:
Shouldn't that be `type: "location/state"` or is that some semi-test to make sure it works in the future?
`{"name":"Test","description":null,"parameters":[{"name":"State","slug":"state","id":"7342756d","type":"location/state"}]}`

If so, can you also make a test with one of the old styles like Product.Category?
`{"name":"Test","description":null,"parameters":[{"name":"Category","slug":"category","id":"8ac835fa","type":"category"}]}`

Reply:
> Shouldn't that be type: "location/state" or is that some semi-test to make sure it works in the future?

That's what I've got directly from observing network requests.

Reply:
Interesting. That's not what I'm getting on release or the CI build. I'm getting the snippets I posted.

Reply:
Just to be sure, I used this exact code BEFORE the fix and could confirm it fails.

Reply:
If it fails before and working now, then merge - I'm just getting tired of these regressions, so I just wanted to test the :turd: of out it, including the old styles to ensure upgrades work.

Reply:
This is from a docker image 0.39.0:
![ ](https://p58.f1.n0.cdn.getcloudapp.com/items/RBuYEBlP/6eb4bee6-0555-4386-b9b4-d4e7e24573e1.gif?source=viewer&v=9607800cfee43a0fbced6766c715e653)

And this is from a local dev on `release-x.39.x`:
https://p58.f1.n0.cdn.getcloudapp.com/items/llu5kg6X/c4f2aadf-64e2-41b0-8d48-d5b28b458a5e.gif?source=viewer&v=d59b575d6dbd1431dd3752eed51fe011

Hence the difference.

Reply:
Exactly. That's 40 style. I don't care anymore - merge it.